source/install/cloud-changelog.md

```diff
@@ -4,12 +4,44 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-01-19](#release-2023-01-19)
 - [Release 2023-01-12](#release-2023-01-12)
 - [Release 2022-12-20](#release-2022-12-20)
 - [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
 - [Release 2022-11-17](#release-2022-11-17)
-- [Release 2022-11-10](#release-2022-11-10)
+
+## Release 2023-01-19
+
+### Improvements
+
+#### User Interface (UI)
+ - Insights and drafts are now included when navigating through channels on the left-hand side using ALT+up/down arrows.
+
+#### Administration
+ - The max size of uploaded emojis is now reduced to 512KiB to reduce image download bandwidth.
+ - Users can now monitor the progress of the bulk export job via its metadata field. It is available at ``mmctl export job show <jobID>``.
+ - Compliance exports no longer timeout when uploading to S3.
+ - Users can now supply a CA file and client certificates for the Elasticsearch client.
```

Comment:
Is the term "CA file" clear to the target audience? (I ask because I'm not 100% sure what a CA file is)